.gitignore

```diff
@@ -7,6 +7,7 @@ bundles/io/org.openhab.io.multimedia.tts.marytts/lib/
 .metadata/
 */plugin.xml_gen
 targetplatform/*.launch
+.settings/
```

Comment:
Because this normally contains IDE settings which I thought are not needed in the repo.
Is this assumption incorrect?